.changes/v2.24.0/657-features.md

```diff
@@ -1,2 +1,2 @@
-Added method `Client.QueryVappNetworks` to retrieve all vApp networks [GH-657]
-Added `VApp` methods `QueryAllVappNetworks`, `QueryVappNetworks`, `QueryVappOrgNetworks` to retrieve various types of vApp networks [GH-657]
+* Added method `Client.QueryVappNetworks` to retrieve all vApp networks [GH-657]
```

Comment:
How about we move this change item to the next release and leave only bug fixes in this one?

Reply:
I think it should be good/useful to have them, what do others think?
@lvirbalas @Didainius

Reply:
I am OK with either choice

Reply:
I'm OK with this, don't want to revert at this point.